crates/re_log_encoding/Cargo.toml

```
@@ -37,7 +37,7 @@ re_smart_channel.workspace = true
 re_tracing.workspace = true
 
 # External:
-ehttp = "0.2"
+ehttp = { git = "https://github.com/emilk/ehttp.git", features = ["streaming"] }
```

```suggestion
ehttp = { git = "https://github.com/emilk/ehttp.git", features = ["streaming"], rev = "884e25a0835ab4cbea53180cb61660100866f28a" }
```

otherwise the build can break when we check out this `rerun` commit months from now (e.g. during a `git bisect`)